sddSchemaMods/1- [sdd].[A2BApplication].sql

```diff
@@ -0,0 +1,98 @@
+BEGIN TRY
```

Comment:
When I run this I get an error.
![image](https://user-images.githubusercontent.com/109524605/205694713-a68ecf16-cb9c-478e-b4f5-603bf22f4a06.png)

Is the intention that this would run as part of the deployment?

Reply:
script already ran against dev DB:-

Reply:
Maybe my db is out of date. Thanks.